### PR TITLE
Kate`s solution

### PR DESCRIPTION
**Pull request by ekaterinakur**
[Preview] (https://ekaterinakur.github.io/js_todo/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
+    <title>Todo</title>
+  </head>
+  <body>
+
+    <script type="text/javascript" src="main.js"></script>
```
Comment (ost-ptk) [inline src/index.html]: use 2 whitespaces instead of 4

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      return true;
+    }
+    
+    return false;
+  }
+
+  getItem(id) {
+    const item = this.todosList.find(todo => todo.id === id);
+
+    if (item) { return item; }
```
Comment (gestap) [inline src/main.js]: https://github.com/mate-academy/style-guides/blob/master/javascript.md#blocks--braces

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: src/main.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+class Todo {
```
Comment (gestap) [inline src/main.js]: use strict mode

File: src/main.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    try {
+      const maxPriorityItem = Math.max(...this.todosList.map(todo => todo.priority));
+      
+      if (!this.todosList.length || maxPriorityItem === NaN) {
+        throw new Error ('Empty array: there are no items in the list');
+      } 
+
+      return this.todosList.find(todo => todo.priority === maxPriorityItem);;
+    } catch(error) {
+      return error.message;
```
Comment (gestap) [inline src/main.js]: you need to `...throw an error` instead return message

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**